### ISSUE TITLE
how to transform the model into .tflite?

### ISSUE THREAD

**Issue by xieshenru** [github-luffy/PFLD-68points-tensorflow#2]

**Comment by xieshenru** [comment]
"F tensorflow/lite/toco/graph_transformations/resolve_batch_normalization.cc:45] Check failed: IsConstantParameterArray(*model, bn_op->inputs[1]) && IsConstantParameterArray(*model, bn_op->inputs[2]) && IsConstantParameterArray(*model, bn_op->inputs[3]) Batch normalization resolution requires that mean, multiplier and offset arrays be constant.",when I transform the model into .tflie,the error  occurred,how to solve this problem? Look forward to your reply.,thanks.

**Comment by epoc88** [comment]
The training does not work without 
./pretrained_models/model.ckpt-51

Any tips?
